cli/src/pcluster/api/controllers/cluster_operations_controller.py

```diff
@@ -69,22 +79,54 @@ def create_cluster(
     :param client_token: Idempotency token that can be set by the client so that retries for the same request are
     idempotent
     """
+    # Set defaults
+    rollback_on_failure = rollback_on_failure or True
+    validation_failure_level = validation_failure_level or ValidationLevel.ERROR
+    dryrun = dryrun or False
     create_cluster_request_content = CreateClusterRequestContent.from_dict(create_cluster_request_content)
-    if create_cluster_request_content.cluster_configuration == "invalid":
-        raise CreateClusterBadRequestException(
-            CreateClusterBadRequestExceptionResponseContent(configuration_validation_errors=[], message="invalid")
+
+    # Validate inputs
+    if client_token:
```

Comment:
I see that we also have this parameter for the `create_image` and `delete_image` APIs, I suppose that we want to raise an exception there as well. How are we planning to use this in the future? Which part of the workflow would need it? I notice that the implementation in the `pcluster_api.py` file does not make use of it

Reply:
we could also remove it from the spec for now. Propagating this token to the CFN create_stack does not seem to have the desired behavior